packages/lib/src/vis/tiles/utils.ts

```diff
@@ -88,11 +88,7 @@ export function getObject3DVisibleBox(
   return NDC_BOX.clone().applyMatrix4(ndcToObject3DMatrix);
 }
 
-export function scaleToLayer(box: Box3, layerSize: Size, meshSize: Size): Box2 {
-  if (box.isEmpty()) {
-    return new Box2();
-  }
-
+export function getLayerScales(layerSize: Size, meshSize: Size) {
```

Comment:
As I needed to convert the `world` coordinates in `layer` coordinates, I extracted the scale generation in this utility